tests/unit/obuf-limits.tcl

```diff
@@ -57,7 +57,7 @@ start_server {tags {"obuf-limits"}} {
         set start_time 0
         set time_elapsed 0
         while 1 {
-            r publish foo bar
+            r publish foo [string repeat x 1000]
```

Comment:
i think we must apply a similar change to the test above us.
otherwise we have no certainty that the fact the client wasn't disconnected is because the time didn't pass, or because we didn't hit the code flow.

Reply:
I did modify the above tests, but I reverted them, because I haven't tested them yet, and today I will test the other two tests accordingly.